examples/sep24/sep24.ts

```diff
@@ -0,0 +1,207 @@
+import axios from "axios";
+import readline from "readline";
+
+import {
+  walletSdk,
+  Anchor,
+  SigningKeypair,
+  Types,
+  IssuedAssetId,
+} from "@stellar/typescript-wallet-sdk";
+import { Memo, MemoText } from "stellar-sdk";
+
+const wallet = walletSdk.Wallet.TestNet();
+const stellar = wallet.stellar();
+const anchor = wallet.anchor({ homeDomain: "testanchor.stellar.org" });
+const account = stellar.account();
+
+let kp: SigningKeypair;
+
+const runSep24 = async () => {
+  const asset = new IssuedAssetId(
+    "USDC",
+    "GBBD47IF6LWK7P7MDEVSCWR7DPUWV3NY3DTQEVFL4NAT4AQH3ZLLFLA5",
+  );
+
+  // Create Account
+
+  const createAccount = async () => {
+    console.log("creating account ...");
+    kp = account.createKeypair();
+    console.log(`kp: \n${kp.publicKey}\n${kp.secretKey}`);
+    await axios.get("https://friendbot.stellar.org/?addr=" + kp.publicKey);
```

Comment:
Can you add a comment that this is for funding the new account

Reply:
https://github.com/stellar/typescript-wallet-sdk/pull/58/commits/3d0558484d2e6ee2a54b2ed2836c999995ef28de